docs/specifications/tests/Nameserver-TP/nameserver01.md

```diff
@@ -1,53 +1,104 @@
-[D.J. Bernstein](http://cr.yp.to/djbdns/separation.html).
+[D.J. Bernstein].
 
-Section 2.5 of [RFC 2870](https://tools.ietf.org/html/rfc2870) have very
+Section 2.5 of [RFC 2870] have very
 specific requirement on disabling recursion functionality on root name
 servers.
 
-### Inputs
+## Inputs
 
-The domain name to be tested.
+* The domain name to be tested ("Child Zone").
 
-### Ordered description of steps to be taken to execute the test case
-1. Retrieve all address records for all the name servers using [Method
-   4](../Methods.md) and [Method 5](../Methods.md).
-2. A SOA query for an almost certainly nonexistent name sent to the each
-   name server IP address found in step 1, with the flag Recursion Desired 
-   (RD) set.
-3. If any answer of the queries made in step 2 contains an RCODE with
-   NXDOMAIN, this test case fails.
+## Ordered description of steps to be taken to execute the test case
 
-### Outcome(s)
+1. Create A queries for the following domain names, or other almost 
+   certainly nonexistent name under different TLDs, with the RD flag
+   set (the names are chosen to violate the 
+   [IDNA 2008 specification]):
+   1. xn--nameservertest.iis.se
+   2. xn--nameservertest.icann.org
+   3. xn--nameservertest.ripe.net
 
-If the response is a possible answer with the RCODE NXDOMAIN, this test
-case fails.
+2. Retrieve all address records for all the name servers for the
+   *Child Zone* using [Method4] and [Method5].
 
-### Special procedural requirements
+3. Repeat the following steps for each retrieved name server IP.
 
-None.
+3. Send the three A queries over UDP.
+
+4. If the name server does not respond with a to a query with a DNS
+   response, then emit *[NO_RESPONSE]*.
+
+5. If the name server sends a response with the RA flag set, then 
+   emit *[IS_A_RECURSOR]*.
+
+6. If the server responds with the RCODE NXDOMAIN in all DNS 
+   responses then emit *[IS_A_RECURSOR]*.
+
+7. If no message has been emitted for the server, then emit 
+   *[NO_RECURSOR]*.
+
+## Outcome(s)
 
-### Intercase dependencies
+The outcome of this Test Case is "fail" if there is at least one message
+with the severity level *ERROR* or *CRITICAL*.
+
+The outcome of this Test Case is "warning" if there is at least one message
+with the severity level *WARNING*, but no message with severity level
+*ERROR* or *CRITICAL*.
+
+In other cases the outcome of this Test Case is "pass".
+
+Message                       | Default severity level (if message is emitted)
+:-----------------------------|:-----------------------------------
+NO_RESPONSE                   | WARNING
+IS_A_RECURSOR                 | ERROR
+NO_RECURSOR                   | INFO
+
+## Special procedural requirements
 
 None.
 
--------
+## Intercase dependencies
+
+If either IPv4 or IPv6 transport is disabled, ignore the evaluation of the
+result of any test using this transport protocol. Log a message reporting
+on the ignored result.
```

Comment:
This paragraph feels out of place here. Either describe how this test case relates to other test cases with regard to disabled transports, or move it to another section (perhaps Special procedural requirements or Steps).